pkg/pillar/hypervisor/kvm.go

```diff
@@ -1148,24 +1393,21 @@ func (f *pciAssignmentsTemplateFiller) pciEBridge(pciID int, pciWOFunction strin
 		return fmt.Errorf("can't write PCIe bridge Passthrough to config file (%w)", err)
 	}
 	f.multifunctionDevices[pciWOFunction].bridgeBus = fmt.Sprintf("pcie-bridge.%d", pciID)
-
 	return nil
 }
 
-func (f *pciAssignmentsTemplateFiller) do(file io.Writer, pciAssignments []pciDevice, pciID int) error {
+func (f *pciAssignmentsTemplateFiller) do(file io.Writer, pciAssignments []pciDevice) error {
```

Comment:
@christoph-zededa, could you please take a look at the changes here as well? I'm looking at it now, but it would be nice if you have a look, too =)